File: app/src/main/java/fr/socket/florian/dhome/view/about/HeaderViewHolder.java
package fr.socket.florian.dhome.view.about;

import android.support.annotation.NonNull;
import android.support.annotation.StringRes;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import fr.socket.florian.dhome.R;

public class HeaderViewHolder extends RecyclerView.ViewHolder {

    private final TextView title;

    public HeaderViewHolder(@NonNull View view) {
        super(view);
        title = view.findViewById(R.id.title);
    }

    public void update(@StringRes int stringRes) {
        title.setText(stringRes);
    }
}

File: app/src/main/java/fr/socket/florian/dhome/view/about/HeaderViewHolder.kt
package fr.socket.florian.dhome.view.about

import android.view.View
import android.widget.TextView
import androidx.annotation.StringRes
import androidx.recyclerview.widget.RecyclerView

import fr.socket.florian.dhome.R

internal class HeaderViewHolder(view: View) : RecyclerView.ViewHolder(view) {

    private val title: TextView = view.findViewById(R.id.title)

    fun update(@StringRes stringRes: Int) { title.setText(stringRes) }
}
